backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/BuiltInFictitiousFunctionIrClassFactory.kt

```diff
@@ -301,7 +304,22 @@ internal class BuiltInFictitiousFunctionIrClassFactory(
 
         fun createFakeOverrideProperty(descriptor: PropertyDescriptor): IrProperty {
             val propertyDeclare = { s: IrPropertySymbol ->
-                IrPropertyImpl(offset, offset, memberOrigin, descriptor, s, descriptor.name)
+                @Suppress("DEPRECATION")
+                /* TODO: [PropertyDescriptor::isDelegated] is deprecated. */
+                IrPropertyImpl(offset,
+                        offset,
+                        memberOrigin,
+                        s,
+                        descriptor.name,
+                        descriptor.visibility,
+                        descriptor.modality,
+                        descriptor.isVar,
```

Comment:
Two many boolean flag arguments, using argument names is highly desirable here.

Reply:
Oh, this is the only not deprecated constructor :/